draft-rpc-rfc7322bis.xml

```diff
@@ -673,41 +673,59 @@ group, the reference entry should include ALL of the RFCs comprising that
 sub-series in a reference grouping under a single citation tag [is it helpful to
 point them to 7991 or the like on how to do this here?].  The authors should
 refer to the specific RFC numbers as part of the text in the body of the
-document and cite the sub series number (for example, "see RFC 2026 of
-[BCP9]").  Inclusion of the URI to the STD or BCP info page (see Section 3.2.3
-of [RFC5741]) is recommended.  The text should appear as follows:</t>
-            <t>See RFC 1034 [STD13].</t>
+document and cite the sub-series number (for example, "see RFC 2026 of
+[BCP9]").  The URI to the STD or BCP info page (see Section 3.2.3 of [RFC5741]) is to be included. The text should appear as follows:</t>
+            <t indent="3">See RFC 3552 [BCP72].</t>
             <t>For an STD or BCP that contains one RFC:</t>
-            <t>[STDXXX]  Last name, First initial., Ed. (if applicable), "RFC
-             Title", Stream, Sub-series number, RFC number, RFC DOI, Date of
-             publication, <eref target="https://www.rfc-editor.org/info/std#">&lt;https://www.rfc-editor.org/info/std#&gt;</eref>.</t>
+            <t indent="3">[STDXXX]  Internet Standard XXX,                              
+<eref
+target="https://www.rfc-editor.org/info/std#">&lt;https://www.rfc-editor.org/info/std#&gt;</eref>. At the time of writing, this STD comprises the following:</t>
+<t indent="3">              Last name, First initial., Ed. (if applicable), "RFC        
+             Title", Stream, Sub-series number, RFC number, RFC DOI, Date of             
+             publication, <eref                                                         
+target="https://www.rfc-editor.org/info/rfc#">&lt;https://www.rfc-editor.org/info/rfc#&gt;</eref>.</t>
+
             <t>Example:</t>
```

Comment:
Should the examples be artwork instead in order to show the hanging indent?

Reply:
I'd like to avoid the formatting update for now.

Reply:
I'd like to leave this for later.  I started to make the updates and it's painful.